SUMMARY.md

```diff
@@ -811,7 +811,7 @@
         * [Use MatcherAssert#assertThat(..)](reference/recipes/java/testing/junit5/usehamcrestassertthat.md)
         * [Use Mockito JUnit Jupiter extension](reference/recipes/java/testing/junit5/usemockitoextension.md)
         * [Use OkHttp 3 MockWebServer for JUnit 5](reference/recipes/java/testing/junit5/upgradeokhttpmockwebserver.md)
-        * [Use Vertx JUnit 5 Extension](reference/recipes/java/testing/junit5/vertxunittovertxjunit5.md)
+        * [Use Vertex JUnit 5 Extension](reference/recipes/java/testing/junit5/vertxunittovertxjunit5.md)
```

Comment:
This refers to https://github.com/eclipse-vertx/vertx-junit5/blob/master/src/main/java/io/vertx/junit5/VertxExtension.java
```suggestion
        * [Use Vert.x JUnit 5 Extension](reference/recipes/java/testing/junit5/vertxunittovertxjunit5.md)
```